packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts

```diff
@@ -80,10 +79,13 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
         // Module context
         this.highlightManager = ctx.highlightManager;
         this.interactionManager = ctx.interactionManager;
-        this.tooltipManager = ctx.tooltipManager;
         this.scene = ctx.scene;
 
-        this.destroyFns.push(ctx.interactionManager.addListener('contextmenu', (event) => this.onContextMenu(event)));
+        const contextState = InteractionState.Default & InteractionState.ContextMenu;
```

Comment:
I may be wrong, but I believe this should be `Default | ContextMenu`?

`Default & ContextMenu === 0` so will never trigger, right?

Reply:
Fixed on branch,